examples/multiband_raster_stream.ipynb

```diff
@@ -0,0 +1,201 @@
+            "File \u001b[0;32m~/git/geoengine-python/env/lib/python3.10/site-packages/xarray/core/merge.py:641\u001b[0m, in \u001b[0;36mmerge_attrs\u001b[0;34m(variable_attrs, combine_attrs, context)\u001b[0m\n\u001b[1;32m    639\u001b[0m             result \u001b[38;5;241m=\u001b[39m compat_dict_union(result, attrs)\n\u001b[1;32m    640\u001b[0m         \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mValueError\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m--> 641\u001b[0m             \u001b[38;5;28;01mraise\u001b[39;00m MergeError(\n\u001b[1;32m    642\u001b[0m                 \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mcombine_attrs=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mno_conflicts\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m, but some values are not \u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    643\u001b[0m                 \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mthe same. Merging \u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mstr\u001b[39m(result)\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m with \u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mstr\u001b[39m(attrs)\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    644\u001b[0m             ) \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01me\u001b[39;00m\n\u001b[1;32m    645\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m result\n\u001b[1;32m    646\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m combine_attrs \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mdrop_conflicts\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n",
+            "\u001b[0;31mMergeError\u001b[0m: combine_attrs='no_conflicts', but some values are not the same. Merging {'tile_idx_y': -1, 'tile_idx_x': -1} with {'tile_idx_y': -1, 'tile_idx_x': 0}"
```

Comment:
Unfortunately, the notebook does not work.

Reply:
what is the error message and did you use the stream_bands backed?

Reply:
You committed the error